pkg/k8scontext/context.go

```diff
@@ -267,14 +265,16 @@ func (c *Context) GetInstanceUpdateStatus(instanceUpdateStatusName string) (*agi
 	return agpool.(*aginstv1beta1.AzureApplicationGatewayInstanceUpdateStatus), nil
 }
 
-// GetProhibitedTarget returns prohibited target with specified name and namespace
-func (c *Context) GetProhibitedTarget(namespace string, targetName string) *prohibitedv1.AzureIngressProhibitedTarget {
```

Comment:
Was `GetProhibitedTarget` not used anywhere ?

Reply:
it was added by me before for testing the crd, it's not used anywhere